docs/celsius-converter-exception/index.md

```diff
@@ -81,89 +52,41 @@ We should probably put °F after the Fahrenheit value so that it's clear to the
 user what unit of measure they're seeing. We can do so using the string
 concatenation operator (`++`):
 
-<!--#prelude#
-let celsius = "1";
-let convert = x => x;
-let _ =
--->
-```reason
-{(celsius |> float_of_string |> convert |> string_of_float) ++ {js| °F|js} |> React.string}
-```
+## String concatenation (++)
+
+<<< Snippets.re#string-concat{4-6}
```

Comment:
This kind of reference to lines sounds like something that could easily break in the future. I guess the workflow when changing snippets would be:
- Change snippet
- Grep for usages of the snippet region and update them
?

Reply:
Yeah, and I have an unofficial rule to never use a snippet twice in one place. It would probably be redundant anyway, but it could easily break the line highlighting if I'm not careful.